automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationDiskSpec.scala

```diff
@@ -108,6 +110,7 @@ class RuntimeCreationDiskSpec
   "create runtime and attach an existing a persistent disk" in { googleProject =>
     val randomeName = randomClusterName
     val runtimeName = randomeName.copy(asString = randomeName.asString + "pd-spec") // just to make sure the test runtime name is unique
+    val runtimeWithDataName = randomeName.copy(asString = randomeName.asString + "pd-spec-data-persist")
```

Comment:
`.copy(asString = randomeName.asString + "pd-spec-data-persist")` isn't actually needed btw :) I added it in line 112 when I was trying to figure out a weird bug..but `randomeName` itself should be able to generate unique names